frontend/src/lib/components/Users/CreateUserModal.svelte

```diff
@@ -0,0 +1,40 @@
+﻿<script lang="ts">
+  import { Modal } from '$lib/components/modals';
+  import t from '$lib/i18n';
+  import { helpLinks } from '$lib/components/help';
+  import CreateUser from '$lib/components/Users/CreateUser.svelte';
+  import Markdown from 'svelte-exmarkdown';
+  import { NewTabLinkRenderer } from '$lib/components/Markdown';
+  import Icon from '$lib/icons/Icon.svelte';
+
+  let createUserModal: Modal;
+  export let endpoint: 'register' | 'acceptInvitation' | 'createGuestUserByAdmin';
+
+  export async function open(): Promise<void> {
+    await createUserModal.openModal(true, true);
+  }
+</script>
+
+<Modal bind:this={createUserModal} bottom>
+  <div class="alert alert-info gap-4 mb-4">
+    <Icon icon="i-mdi-info-outline" size="text-2xl" />
+    <div>
+      <h3 class="text-lg">{$t('common.did_you_know')}</h3>
+      <div>
+        <Markdown
+          md={$t('admin_dashboard.create_user_modal.help_create_single_guest_user', { helpLink: helpLinks.addProjectMember })}
+          plugins={[{ renderer: { a: NewTabLinkRenderer } }]}
+        />
+        <Markdown
+          md={$t('admin_dashboard.create_user_modal.help_create_bulk_guest_users', { helpLink: helpLinks.bulkAddCreate })}
+          plugins={[{ renderer: { a: NewTabLinkRenderer } }]}
+        />
+      </div>
+    </div>
+  </div>
+  <h1 class="text-center text-xl">{$t('admin_dashboard.create_user_modal.create_user')}</h1>
+  <CreateUser {endpoint} autoLogin={false} allowUsernames={true}
```

Comment:
Prettiness 🤩 
```suggestion
  <CreateUser {endpoint} autoLogin={false} allowUsernames
```

Reply:
Done in commit adebdc60 — I'd forgotten Svelte lets you do that with boolean props. Much nicer. Too bad there isn't a form like `!autoLogin`, but that would probably be too much parsing for too little gain for the Svelte team to consider that.